pkg/trace/sampler/remote_rates.go

```diff
@@ -61,20 +65,14 @@ func newRemoteRates(client config.RemoteClient, maxTPS float64, agentVersion str
 	}
 }
 
-func (r *RemoteRates) onUpdate(update map[string]state.APMSamplingConfig) {
+func (r *RemoteRates) onUpdate(updates []RemoteRateUpdate) {
 	// TODO: We don't have a version per product, yet. But, we will have it in the next version.
```

Comment:
It looks like these new remote rate updates have a version field, should we delete this TODO?

Reply:
I'll remove this comment and add one that explains we're removing this remote config stuff

Reply:
done :)